test/unittests/capi_test.cpp

```diff
@@ -330,7 +330,9 @@ TEST(capi, find_exported_function)
     EXPECT_EQ(function.type.output, FizzyValueTypeI32);
     EXPECT_NE(function.context, nullptr);
     ASSERT_NE(function.function, nullptr);
-    EXPECT_THAT(function.function(function.context, instance, nullptr, 0), CResult(42_u32));
+    fizzy::ExecutionContext ctx;
+    const auto c_ctx = reinterpret_cast<FizzyExecutionContext*>(&ctx);
```

Comment:
```suggestion
    const auto* c_ctx = reinterpret_cast<FizzyExecutionContext*>(&ctx);
```

Reply:
This is not the same. We want `auto* const` here.

Reply:
Changed to `auto* const`.